tensorboard/plugins/histogram/summary_v2.py

```diff
@@ -41,6 +41,36 @@
 def histogram(name, data, step=None, buckets=None, description=None):
     """Write a histogram summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`.
+
+    Logs a histogram to the current log directory for later analysis in
+    TensorBoard's 'Histograms' and 'Distributions' dashboards (one histogram
+    will appear in both). Like `tf.summary.scalar` points, each histogram is
+    associated with a `step` and a `name`. All the histograms with the same
+    `name` constitute a time series of histograms.
+
+    This example writes 2 histograms:
+
+    ```python
+    test_summary_writer = tf.summary.create_file_writer('test/logs')
+    with test_summary_writer.as_default():
+      tf.summary.histogram("activations", [-10, 0, 10], step=0)
+      tf.summary.histogram("initial weights", tf.random.normal([5]), step=0)
```

Comment:
I'd consider using a bigger size than 3 and 5 which is A) a bit more realistic and B) will result in a nicer histogram, rather than showing how the histogram smoothing is wonky when there are only a few values.  Something >30 (default number of buckets) would be best.  If we just use `tf.random.normal()` you might as well pass `[1000]` or something so the plot looks pretty.

Reply:
Done.